### ISSUE TITLE
Bug: Debug not implemented for AgileReference

### ISSUE THREAD

**Issue by leonardder** [microsoft/windows-rs#1982]
### Which crate is this about?

Windows

### Crate version

0.39.0

### Summary

The Debug trait is not implemented for AgileReference.

### Toolchain version/configuration

Default host: x86_64-pc-windows-msvc
rustup home:  C:\Users\<user>\.rustup

stable-x86_64-pc-windows-msvc (default)
rustc 1.63.0 (4b91a6ea7 2022-08-08)

### Reproducible example

```rust
#[derive(Debug)]
#[implement(IWTSPlugin)]
pub struct RdPipePlugin(AgileReference<IWTSPlugin>);
```

### Expected behavior

Debug is implemented for AgileReference as it is with many other types, at least com interfaces

### Actual behavior

The following error is generated:
```
error[E0277]: `AgileReference<IWTSPlugin>` doesn't implement `std::fmt::Debug`
```

**Comment by kennykerr** [comment]
Here's a fix: #1986

**Comment by leonardder** [comment]
thanks for taking this so quickly!
